src/OrbitCaptureClient/CaptureEventProcessor.cpp

```diff
@@ -256,6 +256,7 @@ void CaptureEventProcessor::ProcessGpuQueueSubmission(
 }
 
 void CaptureEventProcessor::ProcessThreadName(const ThreadName& thread_name) {
+  // Note: thread_name.pid() is available, but currently dropped.
```

Comment:
Will you carry the pid to the timer in an upcoming PR?

Reply:
It's not in my plans, no, mostly because if I'm not wrong the plan with @dimitry-'s new capture format is that we'll get rid of the "client protos". This change is meant to make sure the pids end up in the protos that have such a field in the saved capture, once we use a single set of protos for both communication and serialization.